### DISCUSSION TITLE
Return JSON on OnRecordAuthRequest

### DISCUSSION THREAD

**Discussion by starredev** [pocketbase/pocketbase#4026] [Q&A]
Is it possible to send custom json towards frontend after logging in? I tried something like

````
onRecordAuthRequest((e) => {
    return e.httpContext.JSON(200, { message: "test" });
})
````

But httpContext is completely empty object

**Comment by ganigeorgiev** [comment]
The above should work if you change the method name to lowercase, aka. `e.httpContext.json(...)` (see https://pocketbase.io/docs/js-routing/#writing-response-body).
Also note that `e.httpContext` is not a plain JS object and the empty string representation is misleading and means that it doesn't have public serializable fields not that it is empty.
